### ISSUE TITLE
Get loss plotting results recorded properly

### ISSUE THREAD

**Issue by 1ceseismic** [kora-edu/kora-edu.github.io#11]
by default, there are too few training steps / too small for data to be recorded; have to change a couple hyperparameters
